src/azure-cli/README.md

```diff
@@ -1,14 +1,15 @@
 
 # Azure CLI (azure-cli)
 
-Installs the Azure CLI along with needed dependencies. Useful for base Dockerfiles that often are missing required install dependencies like gpg.
+Installs the Azure CLI along with needed dependencies. Useful for base Dockerfiles that often are missing required install dependencies like gpg. It will also optionally install the Azure Developer CLI.
 
 ## Example Usage
 
 ```json
 "features": {
         "ghcr.io/devcontainers/features/azure-cli:1": {
-            "version": "latest"
+            "version": "latest",
+            "azure-dev-cli-version": "latest"
```

Comment:
Can you switch this to camelCase to be consistent with other features in this repo? Example: https://github.com/devcontainers/features/tree/main/src/desktop-lite
```suggestion
            "azureDevCliVersion": "latest"
```